datastore/elastic_search/query.py

```diff
@@ -301,11 +312,11 @@ def _run_es_search(connection, **kwargs):
         kwargs:
             Refer https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.Elasticsearch.search
     Returns:
-        dictionary, search results from elasticsearch.ElasticSearch.search
+        dictionary, search results from elasticsearch.ElasticSearch.msearch
     """
     scroll = kwargs.pop('scroll', False)
     if not scroll:
-        return connection.search(**kwargs)
+        return connection.msearch(**kwargs)
```

Comment:
Add a new argument to this function `msearch=False` only do .msearch() when `msearch` is set to True. Pass msearch=True from line 299

if `scroll` is True and `msearch` is True raise ValueError('Scrolling is not supported in msearch mode`)

Reply:
Have we done any benchmarking to be able to set `max_concurrent_searches` and `max_concurrent_shard_requests`. We should figure these out for optimal performance.
See https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html

Reply:
Done